sphinx.go

```diff
@@ -167,41 +187,6 @@ func (hd *HopData) Encode(w io.Writer) error {
 	if _, err := w.Write(hd.ExtraBytes[:]); err != nil {
 		return err
 	}
-
```

Comment:
Shouldn't the `ExtraBytes` also be omitted? Otherwise it's 13 wasted bytes that can't be used by the greater payload.